csrankings-n.csv

```diff
@@ -413,6 +413,7 @@ Nicu Sebe,University of Trento,http://disi.unitn.it/~sebe,tNtjSewAAAAJ
 Niculae Sebe,University of Trento,http://disi.unitn.it/~sebe,tNtjSewAAAAJ
 Niels A. Taatgen,University of Groningen,http://www.ai.rug.nl/~niels,F5FI6uYAAAAJ
 Niels Anne Taatgen,University of Groningen,http://www.ai.rug.nl/~niels,F5FI6uYAAAAJ
+Niels Grabe,University of Göttingen,http://tigacenter.bioquant.uni-heidelberg.de/grabe.niels.html,zkw0tYgAAAAJ

```

Comment:
Listed at Heidelberg and not obviously in CS, either. Please clarify.

Reply:
Recently moved to Uni Göttingen Medical Center; website not updated yet. Primary affiliation is Göttingen. Professorship for Data Analysis and Fusion for Imaging. Will be affiliated with CS PhD program as well. Official university announcement here: https://www.uni-goettingen.de/en/1.+januar+2022+bis+31.+märz+2022/663802.html (in German)